memdown.js

```diff
@@ -207,3 +240,4 @@ MemDOWN.prototype._iterator = function (options) {
 }
 
 module.exports = MemDOWN.default = MemDOWN
+module.exports.MemIterator = MemIterator
```

Comment:
```suggestion
// Exposed for unit tests only
module.exports.MemIterator = MemIterator
```

(To avoid a major release later when we want to remove this export)